app/root.tsx

```diff
@@ -41,6 +41,10 @@ export function Layout({ children }: { children: React.ReactNode }) {
         <meta name='viewport' content='width=device-width, initial-scale=1' />
         <Meta />
         <Links />
+        {
+          // script below fixes FOUC on Firefox
+        }
+        <script>let FF_FOUC_FIX;</script>
```

Comment:
I should probably update the commit message about this - but this is a Firefox bug that makes NO sense but the fix works so whatever.